docs/lms_config.ini

```diff
@@ -0,0 +1,43 @@
+# below are the server-wide settings for documentation
+[help_settings]
+url_base = http://edx.readthedocs.org/projects/open-edx-learner-guide
```

Comment:
@pdesjardins Should the base URL be the new ones? e.g 
http://edx.readthedocs.io/projects/edx-partner-course-staff/en/latest/